bentoml/deployment/serverless/serverless_utils.py

```diff
@@ -24,47 +24,42 @@
 from subprocess import PIPE
 
 from ruamel.yaml import YAML
-from packaging import version
 
+from bentoml.configuration import BENTOML_HOME
 from bentoml.utils import Path
 from bentoml.utils.tempdir import TempDirectory
 from bentoml.utils.whichcraft import which
-from bentoml.exceptions import BentoMLException
+from bentoml.exceptions import BentoMLException, BentoMLMissingDepdencyException
 
 logger = logging.getLogger(__name__)
 
-MINIMUM_SERVERLESS_VERSION = '1.40.0'
 
+SERVERLESS_VERSION = '1.53.0'
 
-def check_serverless_compatiable_version():
-    if which("serverless") is None:
-        raise ValueError(
-            "Serverless framework is not installed, please visit "
-            + "www.serverless.com for install instructions."
-        )
+# We will install serverless package and use the installed one, instead
+# of user's installation
+SERVERLESS_BIN_COMMAND = '{}/node_modules/.bin/serverless'.format(BENTOML_HOME)
 
-    version_result = (
-        subprocess.check_output(["serverless", "-v"]).decode("utf-8").strip()
-    )
-    if "(Enterprise Plugin:" in version_result:
-        slice_end_index = version_result.find(" (Enterprise")
-        version_result = version_result[0:slice_end_index]
-    parsed_version = version.parse(version_result)
-
-    if parsed_version >= version.parse(MINIMUM_SERVERLESS_VERSION):
-        return
-    else:
-        raise ValueError(
-            "Incompatiable serverless version, please install version 1.40.0 or greater"
-        )
+
+def install_serverless_package():
+    if not os.path.isfile(SERVERLESS_BIN_COMMAND):
```

Comment:
We should still run install right? let's say in the next version of BentoML we want to use a newer version of serverless, this will prevent users from updating to new serverless

Reply:
I think if it's already installed the same version, NPM will skip.. I might be wrong tho